dyn_em/module_first_rk_step_part1.F

```diff
@@ -1154,8 +1154,11 @@ SUBROUTINE first_rk_step_part1 (   grid , config_flags              &
      &          ,SF_URBAN_PHYSICS=config_flags%sf_urban_physics                          &
 ! Bep changes end
 ! add tke_pbl, and turbulent fluxes
-     &          ,TKE_PBL=grid%tke_pbl,EL_PBL=grid%el_pbl,WU_TUR=grid%wu_tur              &
-     &          ,WV_tur=grid%wv_tur,WT_tur=grid%wt_tur,WQ_tur=grid%wq_tur                &
+     &          ,TKE_PBL=grid%tke_pbl,DISS_PBL=grid%diss_pbl,TPE_PBL=grid%tpe_pbl       &
+     &          ,TKE_ADV=scalar(ims,kms,jms,P_tke_adv),DISS_ADV=scalar(ims,kms,jms,P_diss_adv) &
+     &          ,TPE_ADV=scalar(ims,kms,jms,P_tpe_adv),PR_PBL=grid%pr_pbl  &
+     &          ,EL_PBL=grid%el_pbl,WU_TUR=grid%wu_tur              &
+     &          ,WV_tur=grid%wv_tur,WT_tur=grid%wt_tur,WQ_tur=grid%wq_tur        
```

Comment:
@andreazonato Need a '&' at the end of the line. Can also line the lines up better.